integration-tests/src/test/groovy/com/okta/sdk/tests/it/ApplicationsIT.groovy

```diff
@@ -516,4 +476,25 @@ class ApplicationsIT extends ITSupport {
         AppUser readAppUser = app.getApplicationUser(appUser2.getId())
         assertThat readAppUser.getCredentials().getUserName(), equalTo("updated-"+user2.getProfile().getEmail())
     }
+
+    @Test
+    void testOAuth2Flow() {
+
+        Client client = Clients.builder()
+            .setOrgUrl("https://java-sdk.oktapreview.com")
```

Comment:
I don't think we should hard code the org url. We should extract it from yaml config/env var.

Reply:
You _shouldn't_ need to configure this at all, the call to `Clients.builder()` should pull everything from the environment).